core/lib/config/src/configs/consensus.rs

```diff
@@ -126,6 +126,9 @@ pub struct ConsensusConfig {
     /// Maximal allowed size of the payload in bytes.
     pub max_payload_size: usize,
 
+    /// View timeout duration in milliseconds.
+    pub view_timeout: usize,
```

Comment:
time::Duration

Reply:
also make it an Option<std::time::Duration>, and implement a getter with a default value

Reply:
also make it an Option\<std::time::Duration\>, and implement a getter with a default value